src/test/kotlin/de/nordgedanken/auto_hotkey/ide/documentation/AhkDocumentationProviderTest.kt

```diff
@@ -0,0 +1,103 @@
+package de.nordgedanken.auto_hotkey.ide.documentation
+
+import com.intellij.openapi.projectRoots.impl.ProjectJdkImpl
+import com.intellij.psi.PsiElement
+import de.nordgedanken.auto_hotkey.AhkBasePlatformTestCase
+import de.nordgedanken.auto_hotkey.ProjectDescriptor
+import de.nordgedanken.auto_hotkey.WithOneAhkSdkAsProjDefault
+import de.nordgedanken.auto_hotkey.lang.core.AhkFileType
+import de.nordgedanken.auto_hotkey.project.settings.defaultAhkSdk
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.kotest.matchers.string.shouldContain
+import org.junit.jupiter.api.Test
+import java.io.File
+
+
+class AhkDocumentationProviderTest : AhkBasePlatformTestCase() {
+
+    override fun getTestDataPath(): String = File("src/test/resources/").absolutePath
```

Comment:
This is common code that's repeated in a few other test classes, so a convenience constant has already been created for this. You can replace the above with:

```kotlin
override fun getTestDataPath() = AhkTestCase.testResourcesPath
```